pkg/ingester/lookupplan/stats.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/prometheus/tsdb"
 	"github.com/prometheus/prometheus/tsdb/index"
-	"github.com/tylertreat/BoomFilters"
+	boom "github.com/tylertreat/BoomFilters"
```

Comment:
Did you change this on purpose? It seems a noop for the rest of the file, and looks unrelated 🤔